src/edu/washington/escience/myria/operator/agg/UserDefinedAggregatorFactory.java

```diff
@@ -123,8 +123,7 @@ public UserDefinedAggregatorFactory(
     }
     resultSchema = new Schema(types, names);
     return ImmutableList.of(
-        new UserDefinedAggregator(
-            initEvaluator, updateEvaluator, emitEvaluators, resultSchema, stateSchema, offset));
+        new UserDefinedAggregator(initEvaluator, updateEvaluator, resultSchema, stateSchema));
```

Comment:
OK, so the `emitEvaluators` are now computed from `emitters` in the `Aggregate` operator?